cloudinit/net/cmdline.py

```diff
@@ -91,7 +91,35 @@ def render_config(self):
         )
 
 
-_INITRAMFS_CONFIG_SOURCES = [KlibcNetworkConfigSource]
+class NetplanConfigSource(InitramfsNetworkConfigSource):
+    """InitramfsNetworkConfigSource for netplan initramfs. """
+
+    def __init__(self, _files=None):
```

Comment:
Just to make sure I'm clear, are you keeping the naming of this parameter with a leading underscore because you are trying to indicate it's not intended to be used anywhere except in testing injection?

Reply:
Correct;  in stages.py the call is to cloudinit.net.cmdline.read_initramfs_config(); the implementation there creates an instance of each class, and calls is_applicable() and then render_config();  so nothing passes _files params into the class in runtime code.